functions/funtions.py

```diff
@@ -104,24 +103,24 @@ def predict(
     """Predict the ratings to movies
 
     Args:
-        test_set_origin (numpyarray)
-        x_id (int)
-        x_rated (numpyarray)
-        S (numpyarray)
-        k (int)
-        min_k (int)
-        x_list (numpyarray)
-        y_list (numpyarray)
-        global_mean (float)
-        true_testset_movies_id (numpyarray)
-        bu (numpyarray)
-        bi (numpyarray)
-        pu (numpyarray)
-        qi (numpyarray)
-        algorithm (str, optional). Defaults to "MF".
+        test_set_origin (numpyarray): storing all user/item pairs we want to predict the ratings.
+        x_id (int): the user_id we want to recommend to him.
+        x_rated (numpyarray): All users who rated each item are stored in list.
+        S (numpyarray): Compute the similarity between all pairs of users.
+        k (int): Number of neighbors use in prediction
+        min_k (int): The minimum number of neighbors to take into account for aggregation. If there are not enough neighbors, the neighbor aggregation is set to zero
+        x_list (numpyarray): All user id in training set.
+        y_list (numpyarray): All movie id in training set.
+        global_mean (float): mean ratings in training set.
+        true_testset_movies_id (numpyarray): list all actual movie id.
+        bu (numpyarray): users biases vector.
+        bi (numpyarray): items biases vector.
+        pu (numpyarray): users latent factor matrix.
+        qi (numpyarray): items latent factor matrix.
+        algorithm (str, optional). Model to make recommendations. Defaults to "MF".
```

Comment:
```suggestion
        algorithm (str, optional). The algorithm used to make recommendations. Possible values are "kNN" or "MF. Defaults to "MF".
```
We should check if the algorithm string is valid or not imo.